.github/workflows/eks-beta-add-on-integ-test.yml

```diff
@@ -76,6 +76,17 @@ jobs:
           role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
           aws-region: us-west-2
 
+      - name: Confirm EKS Version Support
+        run: |
+          echo k8s.io/client-go version $(go list -m k8s.io/client-go)
+          echo version to test against ${{ matrix.arrays.k8sVersion }}
+          echo k8s.io/client-go version is v0.x.y matrix version is 1.x
```

Comment:
can we check not just client-go but the other k8s apis too? These all seem to be the same version

```
        k8s.io/api v0.29.0
	k8s.io/apimachinery v0.29.0
	k8s.io/client-go v0.29.0
	k8s.io/component-base v0.29.0
	k8s.io/kubectl v0.29.0
```

Reply:
When we upgrade 1 we upgrade all of them.
Test 
```
➜  amazon-cloudwatch-agent-operator git:(compare-eks-versions) go get k8s.io/client-go@latest
go: downloading k8s.io/client-go v0.29.2
go: downloading k8s.io/api v0.29.2
go: downloading k8s.io/apimachinery v0.29.2
go: upgraded golang.org/x/mod v0.13.0 => v0.14.0
go: upgraded golang.org/x/tools v0.14.0 => v0.16.1
go: upgraded k8s.io/api v0.29.0 => v0.29.2
go: upgraded k8s.io/apimachinery v0.29.0 => v0.29.2
go: upgraded k8s.io/client-go v0.29.0 => v0.29.2
```

Reply:
Added